packages/console/src/components/TextInput/index.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import React, { forwardRef, HTMLProps } from 'react';
+
+import * as styles from './index.module.scss';
+
+// https://github.com/yannickcr/eslint-plugin-react/issues/2856
+/* eslint-disable react/require-default-props */
+type Props = HTMLProps<HTMLInputElement> & {
+  error?: boolean;
```

Comment:
Use `invalid` instead? `error` gives the impression that it's the actual error message instead of a boolean flag

Reply:
actually it's an error state - how about `isErroneous`

Reply:
I get it, but I never saw `isErroneous` in source code. I think `hasError`, `isInvalid` or just `invalid` would sound better

Reply:
`hasError` will be good